bitnami/apache/README.md

```diff
@@ -175,18 +175,28 @@ The command removes all the Kubernetes components associated with the chart and
+| `metrics.resources.limits`                | The resources limits for the container                                                                                                    | `{}`                      |
+| `metrics.resources.requests`              | The requested resources for the container                                                                                                 | `{}`                      |
+| `metrics.podMonitor.enabled`              | if `true`, creates a Prometheus Operator PodMonitor (also requires `metrics.enabled` to be `true`)                                        | `false`                   |
+| `metrics.podMonitor.namespace`            | Namespace for the PodMonitor Resource (defaults to the Release Namespace)                                                                 | `""`                      |
+| `metrics.podMonitor.interval`             | Interval at which metrics should be scraped.                                                                                              | `""`                      |
+| `metrics.podMonitor.scrapeTimeout`        | Timeout after which the scrape is ended                                                                                                   | `""`                      |
+| `metrics.podMonitor.additionalLabels`     | Additional labels that can be used so PodMonitor will be discovered by Prometheus                                                         | `{}`                      |
+| `metrics.podMonitor.relabelings`          | RelabelConfigs to apply to samples before scraping                                                                                        | `[]`                      |
+| `metrics.podMonitor.metricRelabelings`    | MetricRelabelConfigs to apply to samples before ingestion                                                                                 | `[]`                      |
+| `metrics.prometheusRule.enabled`          | if `true`, creates a Prometheus Operator PrometheusRule (also requires `metrics.enabled` to be `true` and `metrics.prometheusRule.rules`) | `false`                   |
+| `metrics.prometheusRule.namespace`        | Namespace for the PrometheusRule Resource (defaults to the Release Namespace)                                                             | `""`                      |
+| `metrics.prometheusRule.additionalLabels` | Additional labels that can be used so PrometheusRule will be discovered by Prometheus                                                     | `{}`                      |
+| `metrics.prometheusRule.rules`            | Prometheus Rule definitions                                                                                                               | `[]`                      |
```

Comment:
Are you using the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool to automatically sync this with the values metadata?

Reply:
yes, regenereted just now with new parameters.